.github/CODEOWNERS

```diff
@@ -1,13 +1,13 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
 
 #python code owners
-python/            @rapidsai/<repo>-python-codeowners
-python/dask_cudf/  @rapidsai/<repo>-dask-codeowners
+python/            @rapidsai/cucim-python-codeowners
+python/dask_cudf/  @rapidsai/cucim-dask-codeowners
```

Comment:
Would it be possible to add @quasiben and myself to this team?

Reply:
Do we need a cucim-dask-codeowners ?

Reply:
Good point. We can drop this. We may want it later, but can always revisit

Reply:
```suggestion
```